README.md

```diff
@@ -120,3 +116,32 @@ Tweet at me! [@TheRealPomax](http://twitter.com/TheRealPomax) or [@TheRealPomax@
 ## And if I just want to use this?
 
 This code is [MIT licensed](https://raw.githubusercontent.com/Pomax/Font.js/master/LICENSE), do whatever you want with it.
+
+## NodeJS
```

Comment:
NodeJS → Node.js

Reply:
I'd suggest moving this part after `## Why wouldn't I use OpenType.js, or Fontkit?` since the contact/license info seem best as last bits.